src/js/common.js

```diff
@@ -361,6 +361,15 @@ global.hideChecksum = function() {
   modal.style.display = 'none'
 }
 
+global.showHideReleaseNotes = function(notes_id) {
+  var notes_div = document.getElementById(notes_id)
+  if (notes_div.style.display === 'none') {
+    notes_div.style.display = 'inline';
+  } else {
+    notes_div.style.display = 'none';
+  }
```

Comment:
One more small change. The first click will fall through here because `notes_div.style.display` is initially unset. Adding/removing the `softHide` class instead of setting style.display makes the toggle occur on first click rather than second:

```suggestion
  if (notes_div.classList.contains('softHide')) {
    notes_div.classList.remove('softHide');
  } else {
    notes_div.classList.add('softHide');
  }
```

There's a much simpler one-liner `notes_div.classList.toggle('softHide')` but it isn't supported in IE.